framework/src/transfers/MultiAppGeneralFieldTransfer.C

```diff
@@ -1294,29 +1325,33 @@ MultiAppGeneralFieldTransfer::outputValueConflicts(
       else
         target_domain_message = "In target parent app mesh,";
 
-      if (hasToMultiApp() && _to_problems.size() > 1)
-        mooseDoOnce(potential_reasons += "Are multiple subapps overlapping?\n";);
-      received_conflict_string += target_domain_message + " point: (" + std::to_string(p(0)) +
-                                  ", " + std::to_string(p(1)) + ", " + std::to_string(p(2)) +
-                                  "), equi-distance: " + std::to_string(std::get<3>(conflict)) +
-                                  "\n";
+      if (num_outputs < _search_value_conflicts_max_log)
+        received_conflicts_string += target_domain_message + " point: (" + std::to_string(p(0)) +
+                                     ", " + std::to_string(p(1)) + ", " + std::to_string(p(2)) +
+                                     "), equi-distance: " + std::to_string(std::get<3>(conflict)) +
+                                     "\n";
+      else if (num_outputs == _search_value_conflicts_max_log)
+        received_conflicts_string +=
+            "Maximum output of the search for value conflicts has been reached. Further conflicts "
+            "will not be output.\nIncrease 'search_value_conflicts_max_log' to output more.";
     }
     mooseWarning("On rank " + rank_str +
                  ", multiple valid values from equidistant points were "
                  "received for target variable '" +
                  getToVarName(var_index) + "' for " + std::to_string(_received_conflicts.size()) +
                  " target points.\n" + potential_reasons + "Conflicts detected at :\n" +
-                 received_conflict_string);
+                 received_conflicts_string);
   }
 
   if (_local_conflicts.empty() && _received_conflicts.empty())
   {
     if (isParamSetByUser("search_value_conflict"))
       mooseInfo("Automated diagnosis did not detect floating point indetermination in transfer");
-    else if (_to_problems.size() > 10 || _from_problems.size() > 10)
-      mooseInfo("Automated diagnosis did not detect any floating point indetermination in "
-                "transfer. You may consider turning it off using `search_value_conflicts=false` "
-                "to improve performance/scalability.");
+    else if (_to_problems.size() > 10 || _from_problems.size() > 10 || _communicator.size() > 10)
```

Comment:
Do you think only outputting for > 10, is better than outputting > 1? It's not a leading question; I want to hear your thinking on it

Reply:
If we have every run output "please turn me off" the default input will just turn it off. When it s useful stuff
10 is arbitrary though. Maybe the threshold for the cost being unbearable is 2 for some sims, maybe it s 100